static/app/views/releases/thresholdsList/thresholdGroupRows.tsx

```diff
@@ -213,56 +206,45 @@ export function ThresholdGroupRows({
   return (
     <StyledThresholdGroup>
       {Array.from(thresholdIdSet).map((tId: string, idx: number) => {
-        const threshold = editingThresholds[tId] || thresholdsById[tId];
-        let environmentName;
-        if (editingThresholds[tId]) {
-          // editing environment type is String
-          environmentName = editingThresholds[tId].environment;
+        let threshold;
+        let isEditing = false;
+        if (tId in editingThresholds) {
+          isEditing = true;
+          threshold = editingThresholds[tId] as EditingThreshold;
         } else {
-          // Threshold environment type is Environnment
-          environmentName = thresholdsById[tId].environment
-            ? thresholdsById[tId].environment.name
-            : '';
+          threshold = initialThreshold as Threshold;
         }
+
         return (
           <StyledRow
             key={threshold.id}
-            lastRow={idx === thresholdIdSet.size - 1}
-            hasError={threshold.hasError}
+            lastRow={isLastRow && idx === thresholdIdSet.size - 1}
```

Comment:
what does lastRow do here?

Reply:
lastRow here determines whether we should render the bottom border or not (we render a border between ever row)

Reply:
Why isn't index enough to know something is last?

Reply:
Yup

Because each grouping here is actually a single 'row'
(each row includes all the "new trhesholds" being created underneath it)

So - we only want to remove the border on the very last "row" of the very last "group"